src/views/example/order-screen/order-screen.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from "react"
+import { View, Image, ViewStyle, TextStyle, ImageStyle, SafeAreaView, SectionList } from "react-native"
+import { NavigationScreenProps } from "react-navigation"
+import { Screen } from "../../shared/screen"
+import { Text } from "../../shared/text"
+import { Button } from "../../shared/button"
+import { Wallpaper } from "../../shared/wallpaper"
+import { Header } from "../../shared/header"
+import { color, spacing } from "../../../theme"
+import { BulletItem } from "../bullet-item"
+import { Api } from "../../../services/api"
+import { save } from "../../../lib/storage"
+import { inject, observer } from "mobx-react"
+import { RootStore } from "../../../app/root-store";
+import { OrderStoreModel } from "../../../app/stores/order-store";
+import { getRoot } from "mobx-state-tree";
+
+const FULL: ViewStyle = { flex: 1 }
+const CONTAINER: ViewStyle = {
+  backgroundColor: color.transparent,
+  paddingHorizontal: spacing[4],
+}
+const DEMO: ViewStyle = {
+  paddingVertical: spacing[4],
+  paddingHorizontal: spacing[4],
+  backgroundColor: "#5D2555",
+}
+const BOLD: TextStyle = { fontWeight: "bold" }
+const DEMO_TEXT: TextStyle = {
+  ...BOLD,
+  fontSize: 13,
+  letterSpacing: 2,
+}
+const HEADER: TextStyle = {
+  paddingTop: spacing[3],
+  paddingBottom: spacing[5] - 1,
+  paddingHorizontal: 0,
+}
+const HEADER_TITLE: TextStyle = {
+  ...BOLD,
+  fontSize: 12,
+  lineHeight: 15,
+  textAlign: "center",
+  letterSpacing: 1.5,
+}
+const TITLE: TextStyle = {
+  ...BOLD,
+  fontSize: 28,
+  lineHeight: 38,
+  textAlign: "center",
+  marginBottom: spacing[5],
+}
+const TAGLINE: TextStyle = {
+  color: "#BAB6C8",
+  fontSize: 15,
+  lineHeight: 22,
+  marginBottom: spacing[4] + spacing[1],
+}
+
+export interface OrderScreenProps extends NavigationScreenProps<{}> {
+  rootStore?: RootStore
+}
+
+/**
+ * inject finds root store in mobx state tree
+ * observer lets us read things from the store
+ */
+@inject("rootStore")
+@observer 
+export class OrderScreen extends React.Component<OrderScreenProps, {orders: Array<any>, user: any}> {
```

Comment:
A really small thing, but should we specify props to be Array<Object> and Object instead of "any" so that we can't pass in any arbitrary type?

Reply:
Good point @jonathancai11, I added that in the latest commit before merging.